src/components/CardGrid/CardGrid.tsx

```diff
@@ -38,6 +39,7 @@ function CardGrid({
   playlist,
   onCardClick,
   onCardHover,
+  getCardImages,
```

Comment:
There are other ways. It is a prop because the `getShelfItemImages` method relies on a param in the config store. The general convention is to not use stores or controllers inside components directly but to pass them via containers instead.

Do you see alternatives?